techniques/aria/ARIA6.html

```diff
@@ -51,57 +51,52 @@
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
   <p>
-    This technique relates to
-    
-<a href="https://w3c.github.io/wcag/understanding/non-text-content">1.1.1: Non-text Content</a>
-(<strong>Sufficient</strong>).</p>
+    このテクニックは <a href="https://waic.jp/translations/WCAG22/Understanding/non-text-content">1.1.1: 非テキストコンテンツ</a> (<strong>十分なテクニック</strong>) に関連する。</p>
   
 
 
 
-  <p>This technique applies to technologies that support <a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA)</a>.</p>
+  <p>このテクニックは、<a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA)</a> をサポートする技術に適用される。</p>
 
 </div>
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-      <p>The purpose of this technique is to provide a label for objects that can be read by assistive technology. The <code class="language-html">aria-label</code> attribute provides the text label for an object, such as a button. When a screen reader encounters the object, the <code class="language-html">aria-label</code> text is read so that the user will know what it is.</p>
-      <p>Authors should be aware that <code class="language-html">aria-label</code> may be disregarded by assistive technologies in situations where <code class="language-html">aria-labelledby</code> is used for the same object. For more information on the naming hierarchy please consult the <a href="https://www.w3.org/TR/accname/#mapping_additional_nd_te">accessible name and description computation</a> section of the Accessible Name And Description Computation recommendation. Authors should be aware that use of <code class="language-html">aria-label</code> will override any native naming such as <code class="language-html">alt</code> on images or <code class="language-html">label</code> associated with a form field using the <code class="language-html">for</code> attribute.</p>
-   </section><section id="examples"><h2>Examples</h2>
+<section id="description"><h2>解説</h2>
+      <p>このテクニックの目的は、支援技術によって読み取ることができるオブジェクトのラベルを提供することである。<code class="language-html">aria-label</code> 属性は、ボタンなど、オブジェクトのテキストラベルを提供する。スクリーンリーダーがオブジェクトに遭遇した際、<code class="language-html">aria-label</code> テキストが読み込まれ、利用者はそのオブジェクトがどのようなものかを知ることができる。</p>
+      <p>コンテンツ制作者は、<code class="language-html">aria-labelledby</code> が同じオブジェクトに使用される状況では、<code class="language-html">aria-label</code> が支援技術によって無視される場合があることに注意すべきである。名前付けの序列の詳細については、Accessible Name And Description Computation 勧告の <a href="https://www.w3.org/TR/accname/#mapping_additional_nd_te">accessible name and description computation</a> の節を参照。コンテンツ制作者は、<code class="language-html">aria-label</code> の使用が画像の <code class="language-html">alt</code> や <code class="language-html">for</code> 属性を用いてフォームフィールドに関係付けられた <code class="language-html">label</code> などのネイティブな名前付けを全て上書きすることに注意すべきである。</p>
```

Comment:
hierarchy は迷うところですが、「優先度」とするのもありかと思いました（が、序列も誤りではないと思うので、採用するかどうかはお任せします）
https://eow.alc.co.jp/search?q=hierarchy

```suggestion
      <p>コンテンツ制作者は、<code class="language-html">aria-labelledby</code> が同じオブジェクトに使用される状況では、<code class="language-html">aria-label</code> が支援技術によって無視される場合があることに注意すべきである。名前付けの優先度の詳細については、Accessible Name And Description Computation 勧告の <a href="https://www.w3.org/TR/accname/#mapping_additional_nd_te">accessible name and description computation</a> の節を参照。コンテンツ制作者は、<code class="language-html">aria-label</code> の使用が画像の <code class="language-html">alt</code> や <code class="language-html">for</code> 属性を用いてフォームフィールドに関係付けられた <code class="language-html">label</code> などのネイティブな名前付けを全て上書きすることに注意すべきである。</p>
```